Formula/d/duplicity.rb

```diff
@@ -34,7 +34,8 @@ class Duplicity < Formula
   depends_on "python-pyparsing"
   depends_on "python-pytz"
   depends_on "python-typing-extensions"
-  depends_on "python@3.12"
+  # upstream as of 2.1.5 requires a Python no higher than 3.11
+  depends_on "python@3.11"
```

Comment:
Is this a new requirement as of 2.1.5? Seems weird we would have to downgrade if it was working previously.

Reply:
I was also confused at first. The reason was because `duplicity` never intended to support 3.12, but their version check in their setup.py was broken. They fixed it in a commit [here](https://gitlab.com/duplicity/duplicity/-/commit/c6fa239297f2f261a8ff8bc47a098ecfa1f6aa9e) shortly before releasing 2.1.5.